index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header>
+        <nav class="navbar">
+          <div class="title"><a>Welcome</a></div>
+          <img src="/images/icon-menu.png" alt="" />
+        </nav>
+      <section class="hero">
+        <h1 class="first">Hey there. I’m Bogdan</h1>
+        <h1 class="second">I’m a software developer</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </section>
```

Comment:
- [ ] As requested by the previous code reviewer, which is also a requirement for this project, please make sure you set images that are backgrounds in the style sheet as background images. You have simply removed the images, now the background is missing. Please kindly fix that. 
***Your implementation***
![kakaI](https://user-images.githubusercontent.com/65852621/138240421-f4b3f970-91cf-4d17-b057-ef6100209ac8.png)

***The design****
![sakaD](https://user-images.githubusercontent.com/65852621/138240382-51541b66-0ba5-4fd3-ba4e-43d5b1087bde.png)